src/doom/am_map.c

```diff
@@ -79,9 +82,8 @@ extern boolean inhelpscreens; // [crispy]
 #define CDWALLRANGE	YELLOWRANGE
 #define THINGCOLORS	GREENS
 #define THINGRANGE	GREENRANGE
-#define SECRETWALLCOLORS	252 // [crispy] purple
+#define SECRETWALLCOLORS WALLCOLORS // preserve the Choco macro
```

Comment:
Well, if we preserve it unchanged, then we don't need a comment. 😉

Reply:
OK, done :)